src/components/dialog-full-screen/dialog-full-screen.d.ts

```diff
@@ -16,8 +39,10 @@ export interface DialogFullScreenProps extends ModalProps {
   subtitle?: string;
   /** Title displayed at top of dialog */
   title?: React.ReactNode;
+  /** The ARIA role to be applied to the DialogFulscreen container */
+  role?: string;
 }
 
-declare class DialogFullScreen extends Modal<DialogFullScreenProps> {}
+declare function DialogFullScreen(props: DialogFullScreenProps): JSX.Element;
```

Comment:
This line here should fix this open issue. https://github.com/Sage/carbon/issues/4669
Would you be able to add a `fixes` or `closes` message to the commit so we can get this issue closed at the same time please?

Reply:
I wasn't aware of this issue, it's now linked in the commit message